contrib/endpoints/include/version

```diff
@@ -0,0 +1 @@
+1.1.0
```

Comment:
There are two places need version:  service_control/proto.cc and cloud_trace.  
They like to send service agent with version.